bin/GameIndex.yaml

```diff
@@ -35696,7 +35696,7 @@ SLUS-21148:
   region: "NTSC-U"
   compat: 5
 SLUS-21149:
-  name: "Yourself Fitness"
+  name: "Yourself!Fitness"
```

Comment:
missing space after the !

https://capasecovers.files.wordpress.com/2010/01/yourselffitness-ps2.jpg